nconf/bugsnag.go

```diff
@@ -8,6 +8,7 @@ type BugSnagConfig struct {
 	Environment    string
 	APIKey         string `envconfig:"api_key" json:"api_key" yaml:"api_key"`
 	ProjectPackage string `envconfig:"project_package" json:"project_package" yaml:"project_package"`
+	HostNodeName   string `envconfig:"node_name" json:"node_name" yaml:"node_name"`
```

Comment:
Can you add a comment here... something like: 

`If left unset, bugsnag will default to the value returned by os.Hostname`.

Reply:
Donezo!